src/sparql/fix_make_omim_exact.ru

```diff
@@ -23,12 +23,12 @@ WHERE
   # Only make the "exactMatch" assumption of 1:1.
   FILTER NOT EXISTS {
   	?cls oboInOwl:hasDbXref ?value2 .
-    FILTER( STRSTARTS(str(?value2), "OMIM"))
+    FILTER( STRSTARTS(str(?value2), "MIM"))
     FILTER(?value!=?value2)
   }
   
-  FILTER( STRSTARTS(str(?value), "OMIM"))
+  FILTER( STRSTARTS(str(?value), "MIM"))
   FILTER( !isBlank(?cls) && STRSTARTS(str(?cls), "http://purl.obolibrary.org/obo/DOID_"))
-  BIND(IRI(REPLACE(REPLACE(STR(?value), "OMIMPS:", "https://omim.org/phenotypicSeries/PS"), "OMIM:", "https://omim.org/entry/")) as ?iri)
+  BIND(IRI(REPLACE(REPLACE(STR(?value), "MIMPS:", "https://omim.org/phenotypicSeries/PS"), "MIM:", "https://omim.org/entry/")) as ?iri)
```

Comment:
Just dropping some initial thoughts here.

Looking at `component-download-doid.owl`, I see that references like
```owl
        <oboInOwl:hasDbXref>OMIM:PS267700</oboInOwl:hasDbXref>
```

have changed to
```owl
        <oboInOwl:hasDbXref>MIM:PS267700</oboInOwl:hasDbXref>
```

Don't have time at the moment to think deeply about this; I'm not sure if this solution works or not. I can think more about it, but we can also use it and see what happens.

Just dropping that this is related:
- https://github.com/monarch-initiative/omim/issues/111

I suppose that in tandem with this, we have to also make updates to the `omim` repo.

Reply:
Resolving comment - we realized these should just be reverted back to OMIMPS